.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Pull Request
+about: A simple pull request template.
+
+---
+
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
+
+- [ ] Does your PR solve an issue created by a member of staff or event handler?
```

Comment:
I'm not sure whether we need this one, while we do provide a list of issues to fix we welcome other PRs as long as they achieve something meaningful.

Reply:
Sure, I just took the points from the contribution guide.